src/Button.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import './Button.css'
 
-function Button({content, link, colorTheme}) {
+function Button({content, link, colorTheme, clickEvent}) {
   return (
     <>
-      {link && <Link className='button-container' to={link}>
+      {link && <Link onClick={clickEvent} className='button-container' to={link}>
```

Comment:
Are you using the button dynamically ?
Are you redirecting your user to a page based on what button they click ?